webapp/src/app/components/editor/editor.component.ts

```diff
@@ -16,10 +14,12 @@ export class EditorComponent {
 	
 	@ViewChild('sidenavLoadMap', {static: true})
 	sidenavLoadMap!: MatSidenav;
-
-	constructor() {
+	
+	constructor(
+		addEntityMenu: AddEntityMenuService
```

Comment:
Unused dependency injection

Reply:
it needs to be injected somewhere so an instance is created and the service can initialize itself. 
I move the code to an init function and call that to make it more obvious